installer/templates/phx_web/endpoint.ex

```diff
@@ -49,7 +49,14 @@ defmodule <%= endpoint_module %> do
   def init(_key, config) do
     if config[:load_from_system_env] do
       port = System.get_env("PORT") || raise "expected the PORT environment variable to be set"
-      {:ok, Keyword.put(config, :http, [:inet6, port: port])}
```

Comment:
A simpler way to write this would be:

    http = [:inet6, port: port] ++ Keyword.get(config, :http, [])
    {:ok, Keyword.put(config, :http, http)}

Reply:
@josevalim  It seems that a default value of `false` is set for the `:http` key here: https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/endpoint/supervisor.ex#L122.

This prevents me to use `Keyword.get(config, :http, [])` because it will return `false` instead of the expected `[]`.